float64 CAR_WHEELBASE
float64 CAR_DIM_X
float64 CAR_DIM_Y
float64 CAR_DIM_Z
float64 CAR_MASS
float64[] CAR_WHEEL_R

geometry_msgs/Pose world_position
float64 speed
float64 speedKMH

float64 yawRate
float64 accelX
float64 accelY

float64 steerAngle
uint8 brakePedalSts

uint8 actualGear
uint16 RPM
